.github/copilot-instructions.md

```diff
@@ -0,0 +1,57 @@
+# Copilot Instructions for CLI for Microsoft 365
```

Comment:
I wonder if it be worth adding a section like 
`## Useful Context` in which we would specify perfect places to use as context for common operations that we may do in this repo.
For example:
- When creating a new command, always use guidance from https://pnp.github.io/cli-microsoft365/contribute/new-command/build-command-logic as your context

Reply:
Since we describe the process of creating a command in the Examples section, I added some more information there.